lib/matplotlib/axes/_axes.py

```diff
@@ -3367,7 +3367,25 @@ def errorbar(self, x, y, yerr=None, xerr=None,
             property names, *markerfacecolor*, *markeredgecolor*, *markersize*
             and *markeredgewidth*.
 
-            Valid kwargs for the marker properties are `.Line2D` properties:
+            Valid kwargs for the marker properties are:
+
+                - *dashes*
+                - *dash_capstyle*
+                - *dash_joinstyle*
+                - *drawstyle*
+                - *fillstyle*
+                - *linestyle*
+                - *marker*
+                - *markeredgecolor*
+                - *markeredgewidth*
+                - *markerfacecolor*
+                - *markerfacecoloralt*
+                - *markersize*
+                - *markevery*
+                - *solid_capstyle*
+                - *solid_joinstyle*
```

Comment:
```suggestion
            - *dashes*
            - *dash_capstyle*
            - *dash_joinstyle*
            - *drawstyle*
            - *fillstyle*
            - *linestyle*
            - *marker*
            - *markeredgecolor*
            - *markeredgewidth*
            - *markerfacecolor*
            - *markerfacecoloralt*
            - *markersize*
            - *markevery*
            - *solid_capstyle*
            - *solid_joinstyle*
```

(ReStructuredText formatting pecularities.)